test/school_house_web/controllers/post_controller_test.exs

```diff
@@ -26,4 +26,26 @@ defmodule SchoolHouseWeb.PostControllerTest do
       assert body =~ "Page not found"
     end
   end
+
+  describe "filter_by_tag/2" do
+    test "renders a page with blog posts matching the tag", %{conn: conn} do
+      conn = get(conn, Routes.post_path(conn, :filter_by_tag, "general"))
+      body = html_response(conn, 200)
+
+      assert body =~ "Posts tagged \"general\""
```

Comment:
Huh, apparently this needs to be updated to use the html enitity/code for `"` since it is being generated in a string with escaped quotes rather than the template itself:

```suggestion
      assert body =~ "Posts Tagged &quot;general&quot;"
```